app/controllers/new_feature_controller2.rb

```diff
@@ -0,0 +1,7 @@
+class NewFeature2Controller < Devise::SessionsController
+  def hoge
+    a = 1  
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Lint/UselessAssignment: Useless assignment to variable - a.